test/new-e2e/system-probe/config/platforms.json

```diff
@@ -0,0 +1,36 @@
+{
+    "url_base": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs",
+    "x86_64": {
+        "ubuntu_16.04": "master/xenial-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_18.04": "master/bionic-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_20.04": "master/focal-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_22.04": "master/jammy-server-cloudimg-x86_64.qcow2.xz",
+        "ubuntu_23.10": "master/mantic-server-cloudimg-x86_64.qcow2.xz",
```

Comment:
these should be updated to use the new path: `master/ubuntu-<version_num>-x86_64.qcow2.xz`